tensorflow/core/api_def/base_api/api_def_StringLower.pbtxt

```diff
@@ -1,3 +1,13 @@
 op {
   graph_op_name: "StringLower"
+  summary: "Lowercases strings in `input`"
+  description: <<END
+Converts each uppercase character of each string in the input Tensor to
+lowercase.
+
+>>> strings = ['Hello', 'TensorFlow']
+>>> tf.strings.lower(strings).numpy()
```

Comment:
No need to have .numpy() in the documentation, we can print tensors (same below)

Reply:
Ok, solved!

Reply:
By the way, I used `.numpy()` because in other places is used to do the same thing, for example in https://github.com/tensorflow/tensorflow/blob/1e65730120aafc413e8c3dcddcf19cd8d184fe1b/tensorflow/core/api_def/base_api/api_def_StringLength.pbtxt#L30 (https://www.tensorflow.org/api_docs/python/tf/strings/length?version=nightly).